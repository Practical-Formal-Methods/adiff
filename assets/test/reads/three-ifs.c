int main() {
	int x,y;

	if (x) { ; }
	if (x) { ; }
	if (x) { ; }




}
