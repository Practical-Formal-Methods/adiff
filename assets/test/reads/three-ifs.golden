int main()
{
    int x, y, z;
    __VERIFIER_read(x);
    if (x)
    {
        1;
    }
    __VERIFIER_read(y);
    if (y)
    {
        2;
    }
    __VERIFIER_read(z);
    if (z)
    {
        3;
    }
}